stage1/diagexec/diagexec.c

```diff
@@ -221,12 +265,14 @@ int main(int argc, char *argv[])
 		NULL
 	};
 
-	const char *root, *cwd, *env, *uid_str, *gid_str, *exe;
-	char **args;
-	uid_t uid;
-	gid_t gid;
+	const char	*root, *cwd, *env, *uid_str, *gid_str, *exe;
+	char		**args;
+	uid_t		uid;
+	gid_t		*gids;
+	size_t		n_gids;
```

Comment:
Apparently I'm wrong: I read `man setgroups` again, and it expects a `size_t`. Please ignore my last comment.